includes/templates/partials/transfer-box-template.php

```diff
@@ -11,32 +11,32 @@
 
 			<div class="Omise-Element RadioBox SELECTED">
 				<input checked="checked" id="transfer_type_full" name="transfer[type]" type="radio" value="full_amount">
-				<h3 class="Omise-Element RadioBoxLabel">Full transfer</h3>
+				<h3 class="Omise-Element RadioBoxLabel"><?php echo Omise_Util::translate( 'Full transfer' ); ?></h3>
 				<p class="Omise-Element RadioBoxDescription">
-					Transfer the whole available balance (<?php echo OmisePluginHelperCurrency::format( $omise['balance']['currency'], $omise['balance']['available'] ); ?>) to your account
+					<?php printf( Omise_Util::translate( 'Transfer the whole available balance (%s) to your account' ), OmisePluginHelperCurrency::format( $omise['balance']['currency'], $omise['balance']['available'] ) ); ?>
 				</p>
 			</div>
 
 			<div class="Omise-Element RadioBox">
 				<input id="transfer_type_partial" name="transfer[type]" type="radio" value="partial">
-				<h3 class="Omise-Element RadioBoxLabel">Partial transfer</h3>
+				<h3 class="Omise-Element RadioBoxLabel"><?php echo Omise_Util::translate( 'Partial transfer' ); ?></h3>
 				<p class="Omise-Element RadioBoxDescription">
-					Transfer only part of the available balance (<?php echo OmisePluginHelperCurrency::format( $omise['balance']['currency'], $omise['balance']['available'] ); ?>) to your account
+					<?php printf( Omise_Util::translate( 'Transfer only part of the available balance (%s) to your account' ), OmisePluginHelperCurrency::format( $omise['balance']['currency'], $omise['balance']['available'] ) ); ?>
 				</p>
 				<div class="Omise-Element Field">
-					<label class="Omise-Element FieldLabel" for="omise_transfer_amount">Amount</label>
+					<label class="Omise-Element FieldLabel" for="omise_transfer_amount"><?php echo Omise_Util::translate( 'Amount', 'Transfer amount' ); ?></label>
 					<input class="Omise-Element TextField" id="omise_transfer_amount" name="omise_transfer_amount" type="text" placeholder="0">
 				</div>
 			</div>
 
 			<div class="Omise-Element FooterAction">
-				<input id="Omise-TransferCreateAction" class="Omise-Element FooterActionElement button button-primary" data-confirm="Do you really want to create this transfer?" name="commit" type="submit" value="Transfer">
-				<span id="Omise-TransferCancelAction" class="Omise-Element FooterActionElement button action">Cancel</span>
+				<input id="Omise-TransferCreateAction" class="Omise-Element FooterActionElement button button-primary" data-confirm="Do you really want to create this transfer?" name="commit" type="submit" value="<?php echo Omise_Util::translate( 'Transfer' ); ?>">
```

Comment:
@nimid I think this one is used to use with the legacy code (when click submit transfer).
So, we can remove it now. (unused)